### PR TITLE
📝 Added Code Acedemy links in various lists

### PR DESCRIPTION
**Pull request by Rid**
Added links to various lists to the Code Academy catalogue for each language.

Signed-off-by: Rid <rid@cylo.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ Tutorials and Guides:
 - [Real Python](https://realpython.com)
 - [The Hitchhiker’s Guide to Python](https://docs.python-guide.org/)
 - [Ultimate Python study guide](https://github.com/huangsam/ultimate-python)
+- [Code Academy, free Python courses](https://www.codecademy.com/catalog/language/python) - This course is a great introduction to both fundamental programming concepts and the Python programming language.
```
Comment (donutboy929) [inline README.md:120]: Could you shorten the descriptions? Remove unnecessary things like "This course is..." and "You will learn...".

This one could be something like: Introduction to the fundamental programming concepts of Python.

**Review by donutboy929 — CHANGES_REQUESTED** [review]
Shorten descriptions to be more concise please :), I'll update CONTRIBUTING.md to reflect the formatting.

File: README.md (hunk 1 above)
Comment (Rid) [reply README.md:120]: NP, Just this one, or all of them?

Comment (donutboy929) [reply README.md:120]: All of them please

**Review by donutboy929 — APPROVED** [review]

**Review by donutboy929 — APPROVED** [review]

**Pull request merged**